### PR TITLE
Add README.md

### LINKED ISSUE #1: Write README
The idea behind this project is to create an application that makes an Http request periodically on the given endpoint. This will give us more control rather than using crontab or agendajs.

Task: Write a README explaining the project and the installation step

README should include following things:
- [ ] Heading of the project
- [ ] Project explanation
- [ ] Minimum python version requirement (3.6)
- [ ] Setting pipenv in python
- [ ] Installing dependencies using pipenv
- [ ] Running the Django project
- [ ] Explaining Project structure
- [ ] Building the Docker container
- [ ] Running the application as a Docker container using docker-compose

### PR DESCRIPTION
**Pull request by prateek2211**
fix #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Using Docker
+
+Alternatively, the application could be run as docker containers, running the API server and one running postgres.
+
+Install docker from [here](https://docs.docker.com/engine/install/ubuntu/) 
+
+Make sure that docker is properly installed in your system.
+```shell script
+sudo docker run hello-world
```
Comment (jatinkatyal13) [inline README.md]: remove sudo

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+```
+Now run the development server:
+```shell script
+python manage.py runserver
+```
+
+Open `http://localhost:8000/admin` to access admin panel and create jobs with appropriate URLs.
+
+## Project Structure
+* `cron`: Mainly contains models for job, application and log. A cronjob is created as soon as the object of `Job` is created in the database.
```
Comment (jatinkatyal13) [inline README.md]: "Django application that stores and handles Cron Jobs in the databases and synchronizes them with the crontab"

**Review by jatinkatyal13 — COMMENTED** [review]
Looks Good ! Left a few comments

File: README.md (hunk 1 above)
Comment (prateek2211) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (prateek2211) [reply README.md]: done

**Comment by prateek2211** [comment]
Does the pr need to be merged after 15 th , in order to be eligible for bounty ?

**Comment by tushargoyal22** [comment]
It should be made after 15, I didn't make actually, i also had updated Readme locally :). Meanwhile, you made it but not an issue :)

**Comment by jatinkatyal13** [comment]
@prateek2211 Don't worry I'll merge your PR after 15, until then you can make the required changes

**Pull request merged**